docs/sdkx-react-native/tracking.mdx

```diff
@@ -0,0 +1,170 @@
+2. Add the custom metadata `Map` in the map expected in the `Parameters` argument with `customMetadata` as the key.
+
+Pass this Map to the following APIs:
+
+`showConversation` API,
+
+```javascript
+const CIFs = { stock_level: { type: 'number', value: '1505' } };
+ const apiConfig = {
+   customIssueFields: CIFs,
+};
+showConversation(apiConfig);
+```
+
+`showFAQsWithConfig` API,
+```javascript
+const CIFs = { stock_level: { type: 'number', value: '1505' } };
+ const apiConfig = {
+   customIssueFields: CIFs,
+};
+showFAQsWithConfig(apiConfig);
+```
+
+`showFAQSectionWithConfig` API
+```javascript
+const CIFs = { stock_level: { type: 'number', value: '1505' } };
+ const apiConfig = {
+   customIssueFields: CIFs,
+};
+showFAQSectionWithConfig(apiConfig);
+```
+
+## Set Custom Issue Fields {#set-custom-issue-fields}
+
+If you want to set Custom Issue Fields at the time of Issue creation, follow the steps.
+
+1. Initialise a top-level custom issue fields `Map`
+2. Define your custom issue field `Map`
+3. Add the `"type"` and `"value"` for that custom issue field
+   You can find the `type` in this constants class `HelpshiftCustomIssueFieldParameter`
+
+
+Pass this Map to `showConversation` API.
+
+Similarly, you can pass this Map to other APIs (ie. `showConversation`, `showFAQsWithConfig`, `showFAQSectionWithConfig`).
+
+The following are the valid values for the `type` key of a Custom Issue Field.
+
+- NewSingleLineParameter
+- NewMultiLineParameter
+- NewNumberParameter
+- NewCheckboxParameter
+- NewDropdownParameter
+- NewDateParameter
+
+Compatibility table for type and values:
+
+| Type                   | Value    | Comments                                                        |
+| ---------------------- | -------- | --------------------------------------------------------------- |
+| NewSingleLineParameter | string   | Character limit of 255                                          |
+| NewMultiLineParameter  | string   | Character limit of 100,000                                      |
+| NewNumberParameter     | int      |                                                                 |
+| NewDropdownParameter   | string   | Drop-down options should exist for the given Custom Issue Field |
+| NewDateParameter       | DateTime | Epoch time. For example - **Date.now()**                        |
+| NewCheckboxParameter   | boolean  |                                                                 |
+
+<Admonition type="info" title="Note">
+
+On Custom Issue Fields keys & compatibility
+
+- Custom Issue Fields must be created in the Helpshift Dashboard (Settings → Custom Issue Fields), otherwise they will be ignored. Read more [here](https://support.helpshift.com/kb/article/custom-issue-fields/)
+
+</Admonition>
+
+<Admonition type="info" title="Note">
+
+In the **Breadcrumbs** and **Debug logs** APIs, the passing of formatted strings is NOT supported. It is recommended to pass data only in a simple String format.
```

Comment:
This note should be in Breadcrumbs and Debug log section

Reply:
We updated this. But in all other doc it's in same place as it now.